data-prepper-plugins/cloudwatch-logs/src/test/java/org/opensearch/dataprepper/plugins/sink/configuration/CwlSinkConfigTest.java

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 package org.opensearch.dataprepper.plugins.sink.configuration;
```

Comment:
`test` and `main` paths should be the same
`org.opensearch.dataprepper.plugins.sink.configuration`
`org.opensearch.dataprepper.plugins.sink.config;`

Reply:
I have made the change to config for both of these sub-directories.